docs/content/docs/getting-started/use-built-provider.md

```diff
@@ -11,75 +11,180 @@ weight: 40
+
+```hcl
+provider_installation {
+
+  # Developer overrides will stop Terraform from downloading the listed
+  # providers their origin provider registries.
+  dev_overrides {
+      "hashicorp/google" = "<REPLACE-ME>\bin"
+      "hashicorp/google-beta" = "<REPLACE-ME>\bin"
+  }
+
+  # For all other providers, install them directly from their origin provider
+  # registries as normal. If you omit this, Terraform will _only_ use
+  # the dev_overrides block, and so no other providers will be available.
+  direct {}
+}
 ```
+
+Edit the file to replace `<REPLACE-ME>` with the output you saved from the first `echo` command, making sure to keep `\bin` at the end of the path. The full path is required and environment variables cannot be used. For example, `C:\Users\JohnDoe\go\bin` is a valid path for a user called `JohnDoe`.
+
+Finally, save the file.
+
 {{< /details >}}
 
-Once this setup is complete, terraform will automatically use the binaries generated by the `make provider` commands in the `terraform-provider-google` and `terraform-provider-google-beta` repositories instead of downloading the latest versions.
+This CLI configuration file you created in the steps above will allow Terraform to automatically use the binaries generated by the `make build` commands in the `terraform-provider-google` and `terraform-provider-google-beta` repositories instead of downloading the latest versions. All other providers will continue to be downloaded from the public Registry as normal.
 
 ## Build provider
 
 ```bash
 # ga provider
 cd $GOPATH/src/github.com/hashicorp/terraform-provider-google
-make provider
+make build
 
 # beta provider
 cd $GOPATH/src/github.com/hashicorp/terraform-provider-google-beta
-make provider
+make build
 ```
 
-## Cleanup
+## Using Terraform CLI developer overrides
+
+To make Terraform use the configuration file you created, you need to set the `TF_CLI_CONFIG_FILE` environment variable to be a string containing the path to the configuration file ([see the documentation here](https://developer.hashicorp.com/terraform/cli/config/environment-variables#tf_cli_config_file)). The path can be either a relative or absolute path.
 
-To stop using the local provider binary, you can run:
+Assuming that a configuration file was created at `~/tf-dev-override.tfrc`, you can either export the environment variable or set it explicitly for each `terraform` command:
 
 ```bash
-rm -rf ~/.terraform.d/plugins/registry.terraform.io/hashicorp/
+# either export the environment variable for your session
+export TF_CLI_CONFIG_FILE="~/tf-dev-override.tfrc"
+
+# OR, set the environment variable value per command
+TF_CLI_CONFIG_FILE="~/tf-dev-override.tfrc" terraform plan
+```
+
+To check that the developer override is working, run a `terraform plan` command and look for a warning near the start of the terminal output that looks like the example below. It is not necessary to run the terraform init command to use development overrides.
```

Comment:
I'm not able to get this output - I'm using terraform 0.13.7. Is there any chance this is only available in newer versions?

Reply:
Yeah that's the case - it's only available in v0.14 and later. I'll add a note to the docs!

Reply:
I think users of Terraform 0.13.x or earlier won't be able to use developer overrides at all, and in their case filesystem mirrors are the only option available.

I'll highlight this in the comment thread where you @ mentioned Riley about whether filesystem mirrors should be documented.